docs/Explorer.md

```diff
@@ -8,6 +8,24 @@ The Kafka explorer shows configured clusters with their topics, brokers, consume
 
 ![Screenshot-3](assets/screen-3.png)
 
+## Actions
+
+### Copy Label
+
+You can copy label of any tree node in the clipboard with `Keyboard` or `"Copy Label" contextual menu`:
+
+![Copy Label](assets/kafka-explorer-copylabel.png)
+
+### Delete
+
+You can delete cluster, topics and consumer groups with `Keyboard`, `Trashcan icon` or `"Delete" contextual menu`:

```

Comment:
You can delete clusters, topics and consumer groups with the `Delete` (`Cmd+Delete` on Mac) key, the `Trashcan` icon or `"Delete" contextual menu`:

Reply:
fixed